tests/fixtures/rust_project/src/lib.rs

```diff
@@ -14,3 +14,5 @@ mod tests {
         assert_eq!(result, 4);
     }
 }
+
+mod stop;
```

Comment:
I put this at the bottom to avoid messing up existing match line numbers

Really this project doesn't need to be valid/compile I suppose but whatever